projects/storefrontstyles/scss/_versioning.scss

```diff
@@ -1,67 +1,67 @@
 /**
- * The Spartacus team is cautious with introducing style changes in a 
- * minor release, as it would conflict with the [semantic versioning](https://semver.org/) 
+ * The Spartacus team is cautious with introducing style changes in a
+ * minor release, as it would conflict with the [semantic versioning](https://semver.org/)
  * model. To improve the style layer during minor releases, without breaking the versioning,
- * a version flag is used. The version is used to control breaking styles during the 
+ * a version flag is used. The version is used to control breaking styles during the
  * lifecycle of a major version.
- * 
- * The breaking changes are added with the  `forVersion()` mixin. The mixin compares the 
- * from version and (optional) to version, and will only add the mixin content if changes are 
- * part of the stable release, or explicitly requested by the customer. 
- * 
- * An example of the usage of the mixin is given below: 
- * 
+ *
+ * The breaking changes are added with the  `forVersion()` mixin. The mixin compares the
+ * from version and (optional) to version, and will only add the mixin content if changes are
+ * part of the stable release, or explicitly requested by the customer.
+ *
+ * An example of the usage of the mixin is given below:
+ *
  * ```scss
  * @include forVersion(2.1){
  *  // valid when 2.1 is explicit requested or from 3.0 onwards
  * }
  * ```
- * 
- * Customers can opt-in for specific versions by declaring the `$styleVersion` variable or by 
- * using the `$useLatestStyles:true` to always be up-to-date with the latest styles. 
+ *
+ * Customers can opt-in for specific versions by declaring the `$styleVersion` variable or by
+ * using the `$useLatestStyles:true` to always be up-to-date with the latest styles.
  */
 
 /**
-  * The $version indicates the latest style version. This version is used to control the 
-  * semantic versioning scheme of the final CSS, to avoid breaking changes in the style layer. 
-  * This allows to introduce breaking changes during minor version, but only if customers explicitly 
+  * The $version indicates the latest style version. This version is used to control the
+  * semantic versioning scheme of the final CSS, to avoid breaking changes in the style layer.
+  * This allows to introduce breaking changes during minor version, but only if customers explicitly
   * enable this. It can be enabled by setting the `$useLatestStyles` to true, or `$styleVersion` to a
-  * specific version. 
+  * specific version.
   */
 $version: 2.1;
 
 /**
  * The `$styleVersion` can be used to explicitly opt-in to breaking style changes till
- * the given style version. It must contain a floating number, such as `2.1`. 
+ * the given style version. It must contain a floating number, such as `2.1`.
  */
-$styleVersion: 0 !default;
+$styleVersion: 2 !default;
```

Comment:
We can default to `$stableStyleVersion` here, so that we only maintain the minor version. 

It would mean that you must change the order of variables, i'm guessing. perhaps we can move the `$version`  and calculated `$stableStyleVersion` to a higher order meta file?

Reply:
Changed the order of variables, but did not extracted it to a separate file, as it's currently only used in this one.